ipaplatform/suse/tasks.py

```diff
@@ -9,6 +9,7 @@
 
 import logging
 
+from ipapython import ipautil
 from ipaplatform.paths import paths
 from ipaplatform.redhat.tasks import RedHatTaskNamespace
 
```

Comment:
```suggestion
from ipaplatform.paths import paths
from ipaplatform.redhat.tasks import RedHatTaskNamespace
from ipapython import ipautil

```

Reply:
I implemented your suggestion in the latest commit.